components/landing/banner/Banner.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './Banner.module.scss';
+
+import Testimonial from '../../../components/testimonial/Testimonial';
+
+const Banner = () => {
```

Comment:
see comment about changing the name of this from `Banner` to something a little more descriptive